.github/workflows/test-ansible-molecule.yml

```diff
@@ -0,0 +1,40 @@
+name: ansible-molecule
+on:
+  pull_request: {}
+
+jobs:
+  check_docs_only:
+    name: check_docs_only
+    runs-on: ubuntu-18.04
+    outputs:
+      should_skip_tests: ${{ steps.check_docs_only.outputs.skip-tests }}
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - id: check_docs_only
+        run: |
+          REPO_MASTER_REF=$(git show-ref ${{ github.base_ref }} | head -1 | cut -d' ' -f2)
+          echo "::set-output name=skip-tests::$(hack/ci/check-doc-only-update.sh $REPO_MASTER_REF)"
+
+  e2e:
+    name: e2e
+    runs-on: ubuntu-18.04
+    needs: check_docs_only
+    if: needs.check_docs_only.outputs.should_skip_tests != 'true'
```

Comment:
we need to comment this one to see if will works. unless you tested it in your fork. 
The molecule target calls 2 molecule tests. See; https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-ansible-molecule.sh#L34 and https://github.com/operator-framework/operator-sdk/blob/master/hack/tests/e2e-ansible-molecule.sh#L44. 

It maybe is too much for one run in GA..

Reply:
See the pR :https://github.com/operator-framework/operator-sdk/pull/4273/files